Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1566,6 +1566,8 @@ bool RenderElement::isInsideEntirelyHiddenLayer() const
 {
     if (isSVGLayerAwareRenderer() && document().settings().layerBasedSVGEngineEnabled() && enclosingLayer()->enclosingSVGHiddenOrResourceContainer())
         return true;
+    if (enclosingLayer() == nullptr)
```

Comment:
Nit: WebKit style is to check `!enclosingLayer()` instead:
https://webkit.org/code-style-guidelines/#zero-comparison